greet.py

```diff
@@ -1,3 +1,13 @@
+import re
+
+
+def is_name_upper(name, upper_list, title_list):
```

Comment:
the name of the function is a bit misleading, since checking if the name is uppercase is not it's main purpose. I would call it `add_to_names_list` or something similar